### ISSUE TITLE
Post title reverts to `_post_title` 

### ISSUE THREAD

**Issue by JustinSainton** [Enclavely/tailor#164]
### Steps to Reproduce

1. Go to any existing page's edit screen.
1. Change the title and update the page.
1. Tailor the page - make any change and save it.
1. Go back to edit screen, see title is reverted to previous title.

This is due to this function: https://github.com/Enclavely/tailor/blob/fd69c688257500d6b633795c41e0a34097fa7c1d/includes/class-panels.php#L384-L394

I'm unclear at the moment on what the _best_ way to address this. Current working theory is that it probably makes sense to hook into post title changes and update the `_post_title` meta key so they stay synced in both directions.

**Comment by conwaydev** [comment]
To override this we just `remove_all_actions( 'tailor_save_settings' );`, but yeah this is a pain.

**Comment by JustinSainton** [comment]
Definitely! Considered that approach as well, but it seemed prone to unintended consequences. 

Our likely approach here will be something like this in a functionality plugin:

```php
remove_action( 'tailor_save_settings', array( tailor_panels(), 'update_post_title' ) );
```

**Comment by conwaydev** [comment]
oh for sure, that too!
